src/locales/en/translation.json

```diff
@@ -391,10 +391,11 @@
   },
   "persist": {
     "createProfile": {
-      "choosePassword": "Choose a password",
       "passwordMismatch": "Entered password does not match",
-      "repeatPassword": "Re-enter your password",
-      "startPersisting": "Store private keys locally, protected by a password",
+      "repeatPassword": "Confirm your password",
+      "startPersisting": "Store your private keys locally and protect them with a password by creating a profile.",
+      "startPersistingToggle": "Create a profile",
+      "undependableAcknowledge": "I understand this password and profile do not substitute my mnemonic.",
```

Comment:
I understand that neither password or profile serve as a substitution for my mnemonic.

Reply:
cc @donouwens do you want to iterate more on it

Reply:
(i'm merging in current state just to avoid code conflict, and will revisit)